inference/nlp/realtime/llm/bloom_176b/djl_deepspeed_deploy.ipynb

```diff
@@ -0,0 +1,613 @@
+{
```

Comment:
<p>Is this going to be in the python sdk so we can use the typical methods of getting a URI that's for your specific region? I think that would fix the CI errors we're seeing too. (We test notebooks across 16 commercial regions, so this will fail 15/16 times.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>right fixed this </p>

Reply:
<p>for this notebook we are leveraging the boto3 api's. We plan to release another notebook down the line with sagemakerr python sdk</p>